src/hb-ot-layout.cc

```diff
@@ -1949,7 +1949,90 @@ hb_ot_layout_get_baseline (hb_font_t               *font,
 			   hb_tag_t                 language_tag,
 			   hb_position_t           *coord        /* OUT.  May be NULL. */)
 {
-  return font->face->table.BASE->get_baseline (font, baseline_tag, direction, script_tag, language_tag, coord);
+  hb_face_t *face = font->face;
+#define INVALID INT_MAX
+  *coord = INVALID;
```

Comment:
Returning false when couldn't find the value inside the font and INT_MAX in the value when simulation has failed.